### ISSUE TITLE
Drop 32-bit Windows Support

### ISSUE THREAD

**Issue by cryptokat** [semuxproject/semux#853]
Oracle has dropped 32-bit Windows support since Java 9.

Source: https://docs.oracle.com/javase/9/migrate/toc.htm#JSMIG-GUID-59701F80-5BB1-416D-835D-C39A9112FC1E

**Comment by orogvany** [comment]
I didn't think 32 bit worked even with java8.  I thought there's been bugs that it hasn't worked..

**Comment by semuxdev** [comment]
Agreed. Not only windows, support for all 32-bit systems has been dropped.

**Comment by cryptokat** [comment]
The windows binary is still executable on 32-bit Windows and there are still some existing 32-bit clients.
![selection_001](https://user-images.githubusercontent.com/32390172/40268373-b642aa00-5b9e-11e8-954c-3863068c71a8.png)

We will need to add a Lanuch4j option `<runtimeBits>64<runtimeBits>` to completely drop 32-bit windows support.
